### PR TITLE
Resolves #44 Better logging

### LINKED ISSUE #44: Logging changes
- Logging should be set to debug, rather than info.
- Logging should use a better named logger, for more targeted configuration

### PR DESCRIPTION
**Pull request by PatrickDRusk**

### CODE REVIEW

File: flowz/artifacts.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ class AbstractArtifact(object):
     An object that wraps the details for asynchronous access to an artifact.
     """
 
-    logger = logging.getLogger('Artifact')
+    logger = logging.getLogger(__name__)
```
Comment (ethanrowe) [inline flowz/artifacts.py]: I don't think this use of `__name__` means what you think.

It isn't resolving to the name of the class; it resolves to the name of the module.

**Review by ethanrowe — CHANGES_REQUESTED** [review]
I have no problem with the log level changes made, but you should double-check the `__name__` thing I mentioned.

For instance, put this in a python file.  Run it directly.  Then import it in the REPL and call show().  I don't think this is what you want.

``` python
class Flah(object):
    namen = __name__

def show():
    print __name__
    print Flah.namen
    print Flah().namen

if __name__ == '__main__':
    show()
```

**Comment by PatrickDRusk** [comment]
I actually meant for it to be the name of the module, so that it would have flowz.artifacts, allowing for some notion of flowz.* for configuring its logging. It is only incrementally better than the prior "Artifacts", but it satisfies the need of allowing someone to turn the flowz logging on or off via something that has the name flowz in it.

You are welcome to take it further (or remove that portion of the change), but this satisfies my requirements.

**Comment by ethanrowe** [comment]
Yeah, I see your point.  Fine.

**Pull request merged**